airbyte-api/src/main/openapi/config.yaml

```diff
@@ -995,6 +997,38 @@ paths:
                 $ref: "#/components/schemas/HealthCheckRead"
         "404":
           description: Debug info not found
+  /v1/migration/export:
+    post:
+      tags:
+        - migration
+      summary: Export Airbyte Configuration and Data
+      operationId: exportAirbyte
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/x-gzip:
+              schema:
+                $ref: "#/components/schemas/MigrationArchive"
```

Comment:
just `Archive`? or if necessary (if archive is a reserved word), maybe `AirbyteArchive`?